### PR TITLE
[17.0][MIG] partner_stage: Migration to 17.0

### PR DESCRIPTION
**Pull request by yankinmax**

### CODE REVIEW

**Review by ivantodorovich — APPROVED** [review]

File: pandoc-3.1.12.3-1-amd64.deb
Comment (SilvioC2C) [inline pandoc-3.1.12.3-1-amd64.deb:1]: What's this file?

File: partner_stage/__manifest__.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "author": "Open Source Integrators, Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/partner-contact",
     "category": "Sales/CRM",
-    "version": "16.0.1.0.1",
+    "version": "17.0.1.0.1",
```
Comment (SilvioC2C) [inline partner_stage/__manifest__.py]:
```suggestion
    "version": "17.0.1.0.0",
```

**Review by SilvioC2C — APPROVED** [review]
Minor changes requested, pre-approving

File: pandoc-3.1.12.3-1-amd64.deb
Comment (yankinmax) [reply pandoc-3.1.12.3-1-amd64.deb:1]: IDK, it's something new generated with `pre-commit`

File: partner_stage/__manifest__.py (hunk 1 above)
Comment (yankinmax) [reply partner_stage/__manifest__.py]: Good catch indeed

**Review by gurneyalex — APPROVED** [review]

**Pull request merged**